db/migrate/20250421190417_add_needs_kms_rotation_to_encrypted_models.rb

```diff
@@ -0,0 +1,11 @@
+class AddNeedsKmsRotationToEncryptedModels < ActiveRecord::Migration[7.2]
+  def change
+    tables = %i[
```

Comment:
Thoughts on using `ApplicationRecord.descendants_using_encryption.map(&:table_name).uniq` here?

Reply:
oh, huh. I didn't know that was an option, but I see it now. I'll give it a try!